test-case/check-pause-release-suspend-resume.sh

```diff
@@ -121,6 +121,8 @@ rm -rf /tmp/sof-test.lock
 #   catch: Evaluate script and trap exceptional returns
 #   after ms: Ms must be an integer giving a time in milliseconds.
 #       The command sleeps for ms milliseconds and then returns.
+# FIXME: Need to handle more of aplay/arecord output. Need to apply similar fix with
+# check-pause-resume.sh, multiple-pause-resume.sh
```

Comment:
Please also add FIXME: this test was broken by 46dadd0c563e ("Add mutual exclusion and journalctl logging") because it never was compatible with `is_subtest()`

Reply:
Will add